app/components/onboarding_email_form/onboarding_email_form.html.erb

```diff
@@ -18,7 +18,7 @@
       <%= c 'input', field.input_defaults.merge(placeholder: '') %>
     <% end %>
 
-    <input type="hidden" id="token" name="token" value="<%= token %>" />
+    <input type="hidden" id="token" name="token" value="<%= jwt %>" />
```

Comment:
I think the `id` here would have to be `jwt` (instead of `token`) as well, right? Because you expect the token to be passed via the `jwt` param…

But actually, maybe removing the hidden input and just adding the JWT to the form action URL as a query param would be way easier. In the `form` tag helper, we could just update the URL to `onboarding_path(jwt: jwt)`.

Reply:
yep, good call.